assets/js/image-gallery.js

```diff
@@ -345,8 +345,8 @@ const loadLightBoxImages = ( blockScope, lightboxSwiper, pageNum, attr, thumbnai
 		} );
 
 	}
-	if ( blockScope.nextElementSibling.classList.contains( 'spectra-image-gallery__control-lightbox' ) ) {
-		const lightbox = blockScope.nextElementSibling;
+	if ( blockScope?.nextElementSibling?.classList.contains( 'spectra-image-gallery__control-lightbox' ) ) {
+		const lightbox = blockScope?.nextElementSibling;
```

Comment:
```suggestion
	const lightbox = blockScope?.nextElementSibling;
	if ( lightbox && lightbox?.classList.contains( 'spectra-image-gallery__control-lightbox' ) ) {
```
We can modify like this.